js/Discourse.js

```diff
@@ -561,8 +547,9 @@ class Discourse extends React.Component {
                     tabBarBackground: () => this._blurView(theme.name),
                   }}>
                   <Tab.Screen
-                    name={i18n.t('home')}
+                    name="Home"
```

Comment:
I know this was intentional but it's ok to lose i18n?

Reply:
Oh wow I'm an idiot. I see it now.

Reply:
Yeah, this was a pretty stupid regression on my part. It caused some ugly routing errors for users on non-English locales. (That's why there are tests for this now.)